{
  "artifacts": [],
  "command": "main",
  "experiment": {
    "base_dir": "/home/lgrootde/Generative-re-tests",
    "dependencies": [
      "datasets==2.17.1",
      "evaluate==0.4.1",
      "numpy==1.26.4",
      "sacred==0.8.5",
      "torch==2.2.1",
      "transformers==4.38.1",
      "wasabi==1.1.2"
    ],
    "mainfile": "run.py",
    "name": "run",
    "repositories": [
      {
        "commit": "086fc2a430e536c211be5a8047e79058fcfd5f4b",
        "dirty": true,
        "url": "https://github.com/ontox-hu/Generative-re-tests"
      }
    ],
    "sources": [
      [
        "run.py",
        "_sources/run_cfdf63a5c4ed4799c21dc3572aca4f10.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/run.py\", line 248, in main\n    trainer = Seq2SeqTrainer(\n              ^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer_seq2seq.py\", line 56, in __init__\n    super().__init__(\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 440, in __init__\n    raise ValueError(\n",
    "ValueError: You cannot perform fine-tuning on purely quantized models. Please attach trainable adapters on top of the quantized model to correctly perform fine-tuning. Please see: https://huggingface.co/docs/transformers/peft for more details\n"
  ],
  "heartbeat": "2024-03-04T21:19:41.008098",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) Platinum 8358 CPU @ 2.60GHz",
    "gpus": {
      "driver_version": "545.23.08",
      "gpus": [
        {
          "model": "NVIDIA A10",
          "persistence_mode": false,
          "total_memory": 23028
        }
      ]
    },
    "hostname": "jupyternoteboo.src-oraclefreeb.src.surf-hosted.nl",
    "os": [
      "Linux",
      "Linux-5.15.0-1052-oracle-x86_64-with-glibc2.31"
    ],
    "python_version": "3.12.1"
  },
  "meta": {
    "command": "main",
    "config_updates": {},
    "named_configs": [],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2024-03-04T21:19:32.840343",
  "status": "FAILED",
  "stop_time": "2024-03-04T21:19:41.008818"
}